docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+find . \! -user orise -exec chown orise '{}' +
+
+source "/opt/ros/$ROS_DISTRO/setup.bash"
+
+exec gosu orise "$@"
```

Comment:
The result I got is that the "orise" is a superuser, so it can run commands without sudo. Is this the desired behavior? The problem stated in #21 does not happen though.

Reply:
Another comment: the workdir becomes the root '/'

Reply:
`docker exec -it /bin/bash` without specifying user will enter into a shell with root user. Maybe that's what happened.

the desired would be using `docker exec --user orise -it /bin/bash` instead (considering that the container is already started)

Reply:
this resolves my previous comment ffe8b91